sdk/src/Services/MobileAnalytics/Custom/_unity/BackgroundRunner.unity.cs

```diff
@@ -104,6 +104,10 @@ private void DoWork()
                     }
                     Thread.Sleep(BackgroundSubmissionWaitTime * 1000);
                 }
+                catch(ThreadAbortException)
+                {
+                  // handle thread aports more greacefully in unity
```

Comment:
Hi, why not doing below?

catch(ThreadAbortException e) {
    _logger.Error(e, "A thread abort exception occurred in Mobile Analytics Manager.");
}